docs/guides/src/main/server/db.adoc

```diff
@@ -73,57 +72,66 @@ Database schema only accounts for Unicode strings in the following special field
+. Set `oracle.jdbc.defaultNChar` to `true`.
++
+[NOTE]
+====
+Setting this property might negatively impact performance. For more information, see the Oracle JDBC driver configuration documentation.
+====
 
-If the database character set is not Unicode, then to use Unicode characters in the special fields, the JDBC driver needs to be configured with the connection property oracle.jdbc.defaultNChar set to true. It might be wise, though not strictly necessary, to also set the oracle.jdbc.convertNcharLiterals connection property to true. These properties can be set either as system properties or as connection properties. Please note that setting oracle.jdbc.defaultNChar may have negative impact on performance. For details, please refer to Oracle JDBC driver configuration documentation.
+. Optionally, set `oracle.jdbc.convertNcharLiterals` to `true`.
 
-=== Microsoft SQL Server database
+=== Unicode support for a Microsoft SQL Server database
 
-Unicode characters are properly handled only for the special fields. No special settings of JDBC driver or database is necessary.
+Unicode characters are supported only for the special fields for a Microsoft SQL Server database. The JDBC driver and database require no special settings.
 
-=== MySQL database
+=== Configuring Unicode support for a MySQL database
 
-Unicode characters are properly handled provided the database was created with Unicode support in VARCHAR and CHAR fields in the CREATE DATABASE command (e.g. by using utf8 character set as the default database character set in MySQL 5.5. Please note that utf8mb4 character set does not work due to different storage requirements to utf8 character set [1]). Note that in this case, length restriction to non-special fields does not apply because columns are created to accommodate given amount of characters, not bytes. If the database default character set does not allow storing Unicode, only the special fields allow storing Unicode values.
+Unicode characters are supported in a MySQL database if the database was created with Unicode support in the VARCHAR and CHAR fields when using the CREATE DATABASE command. In other words, `utf8` was selected as the default database character set in MySQL 5.5.
 
-At the side of JDBC driver settings, it is necessary to add a connection property characterEncoding=UTF-8 to the JDBC connection settings.
+The utf8mb4 character set is not supported due to different storage requirements for the utf8 character set [1]). In that situation, the length restriction on non-special fields does not apply because columns are created to accommodate the number of characters, not bytes.  If the database default character set does not allow Unicode storage, only the special fields allow storing Unicode values.
```

Comment:
What does the [1] mean? Is it https://dev.mysql.com/doc/refman/8.0/en/alter-table.html#alter-table-character-set or https://issues.jboss.org/browse/KEYCLOAK-3873 that was [in the original text](https://github.com/keycloak/keycloak-documentation/blob/567ca00fe12dbfd22915e85f0cb7f6d53d67cbd9/topics/database/unicode-considerations.adoc)?

```suggestion
The utf8mb4 character set is not supported due to different storage requirements for the utf8 character set, see MySQL documentation for details. In that situation, the length restriction on non-special fields does not apply because columns are created to accommodate the number of characters, not bytes.  If the database default character set does not allow Unicode storage, only the special fields allow storing Unicode values.
```